padatious/util.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 from xxhash import xxh32
+from bracket_expansion import SentenceTreeParser
```

Comment:
To make this work, I needed to change it to `from padatious.bracket_expansion ...`